### PR TITLE
git-machete: fix tests

### PR DESCRIPTION
**Pull request by superherointj**
git-machete: fix tests

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/git-machete/default.nix [hunk 1]
```diff
@@ -1,6 +1,6 @@
 { lib, buildPythonApplication, fetchPypi
 , installShellFiles, pbr
-, flake8, mock, pycodestyle, pylint, tox
+, flake8, mock, pycodestyle, pylint, tox, stestr
```
Comment (figsoda) [inline pkgs/applications/version-management/git-and-tools/git-machete/default.nix:3]:
```suggestion
, flake8, mock, pycodestyle, pylint, stestr, tox
```

File: pkgs/applications/version-management/git-and-tools/git-machete/default.nix [hunk 2]
```diff
@@ -16,10 +16,7 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles pbr ];
 
-  # TODO: Add missing check inputs (2019-11-22):
-  # - stestr
-  doCheck = false;
-  checkInputs = [ flake8 mock pycodestyle pylint tox ];
+  checkInputs = [ flake8 mock pycodestyle pylint tox stestr ];
```
Comment (figsoda) [inline pkgs/applications/version-management/git-and-tools/git-machete/default.nix:19]:
```suggestion
  checkInputs = [ flake8 mock pycodestyle pylint stestr tox ];
```

**Review by figsoda — CHANGES_REQUESTED** [review]
i would sort it like this, otherwise lgtm

File: pkgs/applications/version-management/git-and-tools/git-machete/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/applications/version-management/git-and-tools/git-machete/default.nix]: Done

File: pkgs/applications/version-management/git-and-tools/git-machete/default.nix (hunk 2 above)
Comment (superherointj) [reply pkgs/applications/version-management/git-and-tools/git-machete/default.nix]: Done

**Review by figsoda — APPROVED** [review]

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 138958` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>git-machete</li>
  </ul>
</details>

File: pkgs/applications/version-management/git-and-tools/git-machete/default.nix [hunk 3]
```diff
@@ -16,10 +16,7 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles pbr ];
 
-  # TODO: Add missing check inputs (2019-11-22):
-  # - stestr
-  doCheck = false;
-  checkInputs = [ flake8 mock pycodestyle pylint tox ];
+  checkInputs = [ flake8 mock pycodestyle pylint stestr tox ];
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/git-machete/default.nix:19]: Why do we need tox here?

Comment (figsoda) [reply pkgs/applications/version-management/git-and-tools/git-machete/default.nix:19]: tests seems to be excluded in the pypi version, should we switch the the github version or should we just remove the check inputs

Comment (SuperSandro2000) [reply pkgs/applications/version-management/git-and-tools/git-machete/default.nix:19]: Just switch to the github download. We do that a lot.

Comment (superherointj) [reply pkgs/applications/version-management/git-and-tools/git-machete/default.nix:19]: @figsoda Are you going to make the change in a new PR? How does this work? Also add on checkPhase, `stestr run`. I forgot this. This PR was part of openstack PR that wasn't merged. Wanted to keep the positive changes. But then, I didn't put too much thought into this package, mostly that `stestr` was missing, and is now available.

Comment (figsoda) [reply pkgs/applications/version-management/git-and-tools/git-machete/default.nix:19]: yes

**Pull request merged**